octavia_proxy/api/drivers/elbv2/driver.py

```diff
@@ -139,3 +142,39 @@ def listener_delete(self, session, listener):
         LOG.debug('Deleting listener %s' % listener.to_dict())
 
         session.elb.delete_listener(listener.id)
+
+    def health_monitors(self, session, query_filter=None):
+        LOG.debug('Fetching health monitors')
+
+        query_filter = query_filter or {}
+
+        results = []
+        raw_results = session.elb.health_monitors(**query_filter)
+        for lb in raw_results:
+            result_data = _monitor.HealthMonitorResponse.from_sdk_object(lb)
+            result_data.provider = ELBv2
+            results.append(result_data)
+        return results
+
+    def health_monitor_create(self, session, healthmonitor):
```

Comment:
here you need to check here all incoming enum params like `type` and throw an exception for unsupported in elb2